docs/api/tasks-get-tasks-by-user_id_david.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: page
+---
+
```

Comment:
I removed the "return to landing page" link, because the title of the page provided by the theme provides that link.